components/dropdown/test/dropdown-content.visual-diff.js

```diff
@@ -71,6 +71,7 @@ describe('d2l-dropdown-content', () => {
 			await page.setViewport({ width: 600, height: 500 });
 			const selector = `#${testName}`;
 			await open(page, selector);
+			await new Promise(r => setTimeout(r, 10));
```

Comment:
Added this 10ms for the mobile tests only. There seem to be some rare flakes like https://visual-diff.d2l.dev/screenshots/BrightspaceUI/core/dropdown-content/2021-06-28-15.10.29.73-807132d1a625dff191a065988df4e5c6f962b887-report.html and https://visual-diff.d2l.dev/screenshots/BrightspaceUI/core/dropdown-content/2021-06-28-12.58.33.311-4238a938811ac8fcd921a1ea1f6999a3769eac25-report.html that fail with only a couple of different pixels. Adding the delay was the only solution I found (no failures after 30 re-runs)